src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        return Colors.values()[random.nextInt(Colors.values().length)].toString();
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/1e1cdd32-68cb-40fc-a0fa-7f87eb55577c)

Reply:
In this case, I forgot about the existence of this command, so sorry